modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/macros/TestrayManage.macro

```diff
@@ -101,17 +177,21 @@ definition {
 	}
 
 	macro editNewOptionName {
-		TestrayManage.goToEditPage(valueName = "${valueName}");
+		TestrayContextMenu.clickContextMenuProjects(name = "Environments_");
+
+		TestrayContextMenu.selectItemFromProjectsContextMenu(item = "Edit");
 
 		Type(
 			locator1 = "TestrayManage#NEW_OPTION_NAME",
-			value1 = "${valueType}");
+			value1 = "${key_value}");
 
 		Select(
-			locator1 = "TestrayManage#SELECT_CATEGORY",
-			value1 = "${category}");
+			locator1 = "TestrayManage#SELECT_CATEGORIES_IN_OPTIONS",
+			value1 = "${categorie}");
```

Comment:
Hi @joelrocha68 would you please correct this word? "categorie" must be changed to "category"